lib/sanctify/matcher.rb

```diff
@@ -0,0 +1,27 @@
+module Sanctify
+  class Matcher
+    attr_reader :description, :regex, :config
+    def initialize(description, regex, config)
+      @description = description
+      @regex = regex
+      @config = config
+    end
+
+    def id
+      description.downcase.gsub(' ', '_')
+    end
+
+    def disabled?
```

Comment:
In addition to a `#disabled?` method, it might be nice to an `#enabled?` method as well.

Reply:
I'm happy to add that when there arises a use for it. For now we only call `disabled?` the one time.